packages/intent/src/core/source-policy.ts

```
@@ -10,11 +10,16 @@ import {
 import { readPackageJson } from './package-json.js'
 import { parseSkillSources } from './skill-sources.js'
 import { resolveProjectContext } from './project-context.js'
+import { detectIntentAudience } from '../shared/environment.js'
```

ESLint

[error] `../shared/environment.js` import should occur before import of `./excludes.js`

(import/order)